### PR TITLE
hbase geo repplication

### PR DESCRIPTION
**Pull request by mumian**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@mumian__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jgao). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+          "name": "[variables('clusterNode1').defaultStorageAccount.type]"
+      },
+      "kind": "Storage", 
+      "properties": {}
+    },    
+    {
+      "name": "[variables('clusterNode1').name]",
+      "type": "Microsoft.HDInsight/clusters",
+      "location": "[variables('clusterNode1').location]",
+      "apiVersion": "2015-03-01-preview",
```
Comment (gatneil) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.json]: is there a non-preview api version we can use?

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+          "name": "[variables('clusterNode2').defaultStorageAccount.type]"
+      },
+      "kind": "Storage", 
+      "properties": {}
+    },    
+    {
+      "name": "[variables('clusterNode2').name]",
+      "type": "Microsoft.HDInsight/clusters",
+      "location": "[variables('clusterNode2').location]",
+      "apiVersion": "2015-03-01-preview",
```
Comment (gatneil) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.json]: same

File: 101-hdinsight-hbase-replication-geo/azuredeploy.parameters.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "clusterNamePrefix": {
+          "value": "GEN-UNIQUE-5"
+        },
+        "clusterLoginUserName": {
+            "value": "GEN-USER"
```
Comment (gatneil) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.parameters.json]: GEN-USER doesn't exist; please use GEN-UNIQUE or a generic name like 'azureuser'

File: 101-hdinsight-hbase-replication-geo/azuredeploy.parameters.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        },
+        "clusterLoginPassword": {
+          "value": "GEN-PASSWORD"
+        },
+        "sshUserName": {
+            "value": "GEN-USER"
```
Comment (gatneil) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.parameters.json]: same

**Comment by gatneil** [comment]
a few small comments; otherwise looks good :)

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json (hunk 1 above)
Comment (mumian) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.json]: it is still in preview

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json (hunk 2 above)
Comment (mumian) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.json]: That is the latest version.

File: 101-hdinsight-hbase-replication-geo/azuredeploy.parameters.json (hunk 3 above)
Comment (mumian) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.parameters.json]: Done

File: 101-hdinsight-hbase-replication-geo/azuredeploy.parameters.json (hunk 4 above)
Comment (mumian) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.parameters.json]: Done

**Comment by mumian** [comment]
All changes have been implemented.  Can you take a look?

**Comment by mumian** [comment]
All changes have been implemented.

**Comment by gatneil** [comment]
looks good; merging

**Pull request merged**